<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>337</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>헌금자 개별이름을 사용하지 않는 헌금</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>551</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>사용자 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>88</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>헌금명칭</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="registed_not_use_hun_tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>125</y>
     <width>141</width>
     <height>192</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <column>
    <property name="text">
     <string>기존 등록된 헌금</string>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
   </column>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>115</y>
     <width>551</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>315</y>
     <width>551</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="new_not_use_hun_tableWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>125</y>
     <width>141</width>
     <height>192</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <column>
    <property name="text">
     <string>신규등록 </string>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="gubunsave_button">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>130</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>저장(F12)</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gubuncancel_button">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>180</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>닫기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gubun_view_button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>240</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>다시 보기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selected_row_delete_button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>280</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>선택 행 삭제</string>
   </property>
  </widget>
  <widget class="QComboBox" name="not_use_hun_combo_widget">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>81</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>88</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>회계 구분</string>
   </property>
  </widget>
  <widget class="QComboBox" name="gubun_comboBox_widget">
   <property name="geometry">
    <rect>
     <x>126</x>
     <y>82</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>gubun_view_button</tabstop>
  <tabstop>gubunsave_button</tabstop>
  <tabstop>selected_row_delete_button</tabstop>
  <tabstop>gubuncancel_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>gubun_view_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>registed_data()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>442</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selected_row_delete_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>delete_selected_row()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>452</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gubunsave_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>file_save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gubuncancel_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>file_save_cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>add_not_use_hun_cancel()</slot>
  <slot>file_save()</slot>
  <slot>file_save_cancel()</slot>
  <slot>registed_data()</slot>
  <slot>delete_selected_row()</slot>
 </slots>
</ui>
